apps/taxes/src/modules/app/webhook-response.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiResponse } from "next";
+import { Logger } from "pino";
+import { createLogger } from "../../lib/logger";
+
+export class WebhookResponse {
+  private logger: Logger;
+  constructor(private res: NextApiResponse) {
+    this.logger = createLogger({ event: "WebhookResponse" });
+  }
+
+  failureNoRetry(error: string) {
+    this.logger.error({ error }, "failureNoRetry called with:");
```

Comment:
This is not error log. Error means that we are alerted on emails and slack and we need to fix the app. This is *expected* behavior. So its a debug message. 

`failureRetry` where provider failed - this is error (or even critical/fatal)